tICA/scripts/ComputeGroupTICA.m

```diff
@@ -184,8 +185,8 @@ function ComputeGroupTICA(StudyFolder, SubjListName, TCSListName, SpectraListNam
 
         
         nameParamPart = ['_' num2str(tICAdim) '_' nlfunc IT];
-        ciftisavereset(tICAMaps, [OutputFolder '/' tICAMapsNamePart nameParamPart '.dscalar.nii'], wbcommand);
-        ciftisavereset(tICAVolMaps, [OutputFolder '/' tICAVolMapsNamePart nameParamPart '.dscalar.nii'], wbcommand);
+        cifti_write(tICAMaps, [OutputFolder '/' tICAMapsNamePart nameParamPart '.dscalar.nii']);
+        cifti_write(tICAVolMaps, [OutputFolder '/' tICAVolMapsNamePart nameParamPart '.dscalar.nii']);
```

Comment:
This appears to expect the tICA mixing matrix to be square:

https://github.com/Washington-University/HCPpipelines/blob/308aada226ba1ae54e9444db64e52ca89320eef0/tICA/scripts/ComputeGroupTICA.m#L145-L146

Reply:
Moot for now because this script generates the mixing matrix, and doesn't currently support reducing the number of components compared to ICA (and the sICA components were already removed).